src/sentry/static/sentry/app/views/settings/project/projectOwnership/index.jsx

```diff
@@ -114,6 +114,31 @@ class ProjectOwnership extends AsyncView {
             ]}
           />
         </Form>
+
+        <Form
+          apiEndpoint={`/projects/${organization.slug}/${project.slug}/ownership/`}
```

Comment:
Cramer made a comment about where we save this option, would it make more sense to save it as a project option instead of on the ownership endpoint?

Reply:
Using a project option would avoid needing a schema change which is nice. I am not clear on what guides what data ends up in project options. I will ask around.